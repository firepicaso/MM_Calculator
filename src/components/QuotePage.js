import Quote from "./Quote";

export default function QuotePage() {
    retrun (
        <div className="quote-container">
            <Quote />
        </div>
    )
};