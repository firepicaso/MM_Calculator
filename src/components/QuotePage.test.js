import { render, screen } from '@testing-library/react';
import QuotePage from './QuotePage';
import '@testing-library/jest-dom/extend-expect';

test('testing QuotePage component', () => {
  const { container } = render(<QuotePage />);
  expect(container.firstChild).toMatchSnapshot();
});

test('simulating user interaction', () => {
  render(<QuotePage />);
  const quoteElement = screen.getByText('Loading the Quote');
  expect(quoteElement).toBeInTheDocument();
});
