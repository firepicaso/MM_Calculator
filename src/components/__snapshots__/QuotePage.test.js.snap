// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing QuotePage component 1`] = `
<div
  class="quote-container"
>
  <div
    class="loading"
  >
    Loading the Quote
  </div>
</div>
`;
